op-chain-ops/cmd/check-l2/main.go

```diff
@@ -264,6 +264,41 @@ func checkPredeployConfig(client *ethclient.Client, name string) error {
 			if err := checkEAS(p, client); err != nil {
 				return err
 			}
+
+		case predeploys.SafeL2Addr:
+			if err := checkPredeployBytecode(p, "SafeL2", client, []byte(bindings.SafeL2ABI)); err != nil {
+				return err
+			}
+
+		case predeploys.MultiSendAddr:
```

Comment:
You can see in the autogenerated code that `xxxABI` is deprecated in favor of `xxxMetaData.ABI`

```

// MultiSendABI is the input ABI used to generate the binding from.
// Deprecated: Use MultiSendMetaData.ABI instead.
```